src/ore/src/str.rs

```diff
@@ -291,6 +291,25 @@ impl fmt::Display for Indent {
     }
 }
 
+/// Used to serialize fields of Maps whose key type is not a native string. Annotate the field with
+/// `#[serde(serialize_with = mz_ore::str::map_key_to_string)]`.
+#[cfg(feature = "serde")]
+pub fn map_key_to_string<'a, I, K, V, S>(map: I, serializer: S) -> Result<S::Ok, S::Error>
```

Comment:
Nice, this will be very useful.

Reply:
`ore::str` seems like a slightly weird place to put this! Can you make an `ore::serde` module instead and put it in there?

Reply:
Done.